packages/color-wheel/README.md

```diff
@@ -1,64 +1,286 @@
+        </sp-table-row>
+        <sp-table-row>
+            <sp-table-cell>HSLA</sp-table-cell>
+            <sp-table-cell><code>hsla(0, 100%, 50%, 1)</code>, <code>hsla(150, 100%, 33%, 0.5)</code></sp-table-cell>
+            <sp-table-cell>HSL + Alpha channel (0-1)</sp-table-cell>
+        </sp-table-row>
+        <sp-table-row>
+            <sp-table-cell>HSV</sp-table-cell>
+            <sp-table-cell><code>hsv(0, 100%, 100%)</code>, <code>hsv(150, 100%, 67%)</code></sp-table-cell>
+            <sp-table-cell>Hue (0-360°), Saturation, Value</sp-table-cell>
+        </sp-table-row>
+        <sp-table-row>
+            <sp-table-cell>HSVA</sp-table-cell>
+            <sp-table-cell><code>hsva(0, 100%, 100%, 1)</code>, <code>hsva(150, 100%, 67%, 0.5)</code></sp-table-cell>
+            <sp-table-cell>HSV + Alpha channel (0-1)</sp-table-cell>
+        </sp-table-row>
+        <sp-table-row>
+            <sp-table-cell>Named Colors</sp-table-cell>
+            <sp-table-cell><code>red</code>, <code>rebeccapurple</code>, <code>darkseagreen</code></sp-table-cell>
+            <sp-table-cell>CSS color keywords (<a href="https://developer.mozilla.org/en-US/docs/Web/CSS/named-color">full list</a>)</sp-table-cell>
+        </sp-table-row>
+    </sp-table-body>
+</sp-table>
+</br>
+
+#### Step Size
+
+The `step` attribute controls the increment of hue adjustment when using keyboard navigation. It defines how many degrees the hue changes with each arrow key press:
 
 ```html
-<sp-color-wheel disabled></sp-color-wheel>
+<!-- Fine control: 1 degree per key press (default) -->
+<sp-color-wheel step="1"></sp-color-wheel>
+
+<!-- Medium control: 10 degrees per key press -->
+<sp-color-wheel step="10"></sp-color-wheel>
+
+<!-- Coarse control: 45 degrees per key press (8 stops around the wheel) -->
+<sp-color-wheel step="45"></sp-color-wheel>
+```
+
+The step size affects keyboard navigation:
+
+- Regular arrow keys move by the step value
+- <kbd>Shift</kbd> + arrow keys move by 10× the step value
+- Choose your step size based on your use case:
+    - **step="1"**: Precise color selection, best for professional design tools
+    - **step="10"**: Balanced control, good for general use
+    - **step="45"**: Quick selection between major hues, ideal for simple color pickers
+
+#### Label
+
+Provide a custom aria-label for accessibility:
+
+```html
+<sp-color-wheel label="Select color hue"></sp-color-wheel>
 ```
 
-## Variants
+#### Direction
 
-### Sized
+The color wheel supports both left-to-right and right-to-left layouts:
 
-An `<sp-color-wheel>`’s size can be customized appropriately for its context. By default, the size is size-2400 (192 px on desktop, 240 px on mobile).
+```html
+<sp-color-wheel dir="rtl"></sp-color-wheel>
+```
+
+### Custom Sizing
+
+An `<sp-color-wheel>`'s size can be customized appropriately for its context. By default, the size is size-2400 (192 px on desktop, 240 px on mobile).
```

Comment:
I'm not seeing a dynamic variable being used called `size-2400`. I'm getting a hard-coded `192px` for all breakpoints. 

We do expose mod tokens for height and width: `var(--mod-colorwheel-width)` `var(--mod-colorwheel-height)` but those two values should be the same to keep a perfect circle. the CSS is reusing the width value in both height and width but then expose two different mod tokens to overwrite the CSS attributes. 

Should we show both examples of custom styling? one with inline styles and one using the mod tokens?